index.js

```diff
@@ -40,6 +41,17 @@ app.post('/like', (request, response) => {
   });
 });
 
+app.get('/api/search/*', (req, res) => {
+  var results = userStore.searchUsers(req.params[0]);
+
+  console.log(chalk.green('RESULTS ARE:'));
+  console.log(results);
+  
+  res.json(results);
+});
+
+
+
 app.listen(3000, () => {
   console.log('Web Server is running on port 3000');
 });
```

Comment:
Don't you need to make a reference to your custom.js file in your index.js file?